source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -202,6 +215,65 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol, SelectActionHa
         }
     }
     
+    func increaseIntrinsicContentSize(_ view: NSView) {
+        let key = String(format: "%p", view)
+        subviewIntrinsicContentSizeCollection[key] = NSValue(size: view.intrinsicContentSize)
+    }
+    
+    func decreaseIntrinsicContentSize(_ view: NSView) {
+        // This Empty function design for override methods
+    }
+    
+    func updateIntrinsicContentSize() {
+        // This Empty function design for override methods
+    }
```

Comment:
Instead of making overridable methods
Protocols will be preferable right?

Reply:
we can make with protocol, after this method will be necessary for adaption. this one is class methods, this are not fall down in necessary method. and here our vision is inherit methods and property from superclass to it's child class.
NSView
    |--->  ACRContentStackView
                    |---> ACRContainerView